apps/marketing/content/blog/why-i-started-documenso.mdx

```diff
@@ -0,0 +1,67 @@
+
+Aside from the personal dimension, I had a pretty clear mindset of what I was looking for. The criteria I go on describing happend to click into place one after another, in no particular order. Having experienced no market demand and a very grindy market, I was looking for something more fundamental. Something basic, infrastructure-like, with a huge demand. A growing market, deeply rooted in the growing digitalization of the world.
+
+And to be honest, I just always liked digital signature tools. It’s a product, easy enough to comprehend and build but complex and impactful enough to satisfy a hard need. It’s a product you can build very product-driven since the market and domain are well understood at this point. So when asked about what’s next for me, I literally said “digital, um, let’s say… signatures”. As it turns out, my first gut feeling was spot on, but how spot on I only realized when I started researching the space. An open source document signing company happens to be the perfect intersection of all criteria and personal preferences I described above, it’s pretty amazing actually:
+
+- The global signing market is huge and rapidly growing
+- The signing space is huge dominated by one outdated player, to put it bluntly. Outdated in terms of tech, pricing and ecosystem
+- The signing space is also ridiculously opaque for a space that is based on open web tech, open encryption tech and open signing standards. Even by closed source standards
+- We are currently seeing a renaissance for commercial open source startups, combining venture founder financial with open source mechanics
+- Rebuilding a fundamental infrastructure as open source with a meaningful scale, has a profoundly transformative effect for a space
+- Working in open source requires you to be open, cooperative and inclusive. It also requires quite a bit of context jumping, “going with the flow” and empathy
+- Apart from fixing the signing space, making Documenso successful, would be another domino tile toward open source eating the world, which is great for everyone
+
+Building a company is so complex, it can’t be planned out. Basing it on great fundamentals and the expected dynmamics it the best founders can do in my humble opinion. After these fundamental decisions you are (almost) just along for the ride and need to focus on solving the “convential” problems of starting a company the best you can. With digital signatures hitting so many point of my personal and professional checklist, this already was a great fit. What got me exited at first though, apart from the perspective of drinking caffeine and coding, was this:
```

Comment:
```suggestion
Building a company is so complex, it can’t be planned out. Basing it on great fundamentals and the expected dynamics it is the best founders can do in my humble opinion. After these fundamental decisions, you are (almost) just along for the ride and need to focus on solving the “conventional” problems of starting a company the best you can. With digital signatures hitting so many points of my personal and professional checklist, this already was a great fit. What got me excited at first though, apart from the perspective of drinking caffeine and coding, was this:
```